### PR TITLE
Update modules not in scope, auth and Satellite registration for frh

### PR DESCRIPTION
**Pull request by aneelac22**
Fixing some styles on Satellite Page but, other than that it is ready

### CODE REVIEW

File: src/bootstrap.tsx
Comment (Hyperkid123) [inline src/bootstrap.tsx:61]: I see some old comment

File: src/bootstrap.tsx
Comment (Hyperkid123) [inline src/bootstrap.tsx:89]: Should we trigger this even in non-fedramp env?

File: src/bootstrap.tsx
Comment (Hyperkid123) [inline src/bootstrap.tsx:119]: I think we want to slightly enhance the condition so we can also get the placeholder component in.

```
if(fedRampEnv) {
  return isFedRampReady && modules && scalprumConfig ? <RootApp config={scalprumConfig} /> : <AppPlaceholder />
}
```

File: src/bootstrap.tsx
Comment (Hyperkid123) [inline src/bootstrap.tsx:124]: We will have to be very careful about the potentially missing Provider.

File: src/cognito/auth.ts
Comment (Hyperkid123) [inline src/cognito/auth.ts:27]: Is the `REFRESH_TOKEN` set by the cognito or did we pick that specific key name? I would to have a slightly more unique local storage key if possible.

File: src/cognito/auth.ts
Comment (Hyperkid123) [inline src/cognito/auth.ts:74]: We should probably store the storage keys in some constants so we have them in one place if we have to change them later on.

File: src/components/Header/Header.tsx
Comment (Hyperkid123) [inline src/components/Header/Header.tsx:74]: Is the favorite link supposed to be hidden on fedramp?

File: src/components/RootApp/RootApp.tsx
Comment (Hyperkid123) [inline src/components/RootApp/RootApp.tsx:113]: The IDP checker is required for normal environments. If it's problematic in fedramp it can be disabled, but it has to stay in the rest of the environments.

File: src/jwt/jwt.ts
Comment (Hyperkid123) [inline src/jwt/jwt.ts:243]: I assume these are supposed to be promises?

File: src/components/RootApp/RootApp.tsx
Comment (aneelac22) [reply src/components/RootApp/RootApp.tsx:113]: I did local build and this seems to be not needed in stage either, keith told me it was added for fr in the first place

Comment (Hyperkid123) [reply src/components/RootApp/RootApp.tsx:113]: OK. If Keight says that then its probably true.

**Comment by Hyperkid123** [comment]
@jjaquish This one is quite big and has the potential to break the whole console. Do you think we can run your IQE on this PR before we merge it?

**Comment by Hyperkid123** [comment]
@aneelac22 I tested the PR in UI and it seems to run OK. But I would like to wait for some feedback from Jack. Is it ok of we wait for a bit on this one?

**Comment by aneelac22** [comment]
> @aneelac22 I tested the PR in UI and it seems to run OK. But I would like to wait for some feedback from Jack. Is it ok of we wait for a bit on this one?

Ideally I would like for it to be merged today so, I can reference actual build reference and lock the build to that commit and there are no surprise changes that will effect fr env

**Review by Hyperkid123 — APPROVED** [review]

**Pull request merged**
